src/components/FuelSavingsForm.js

```diff
@@ -32,48 +32,48 @@ class FuelSavingsForm extends React.Component {
         <h2>Fuel Savings Analysis</h2>
         <table>
           <tbody>
-          <tr>
```

Comment:
@nickytonline & @coryhouse 
I believe the white spacing change I made here is correct. Should not a `<tr>` appear as a child of a `<tbody>` in this case and not a sibling?  Please confirm you'd like it to appear as a sibling and I'll revert.

Reply:
Point taken. You can leave it.

Reply:
@coryhouse thanks for the confirmation.